truss/tests/test_truss_handle.py

```diff
@@ -24,6 +26,38 @@ def test_server_predict(custom_model_truss_dir_with_pre_and_post):
     assert resp == {'predictions': [4, 5, 6, 7]}
 
 
+def test_readme_generation_int_example(custom_model_truss_dir_with_pre_and_post):
+    sc = TrussHandle(custom_model_truss_dir_with_pre_and_post)
+    readme_contents = sc.generate_readme()
+    readme_contents = readme_contents.replace('\n', '')
+    readme_correct_path = Path(__file__).parent.parent / 'test_data' / 'readme_int_example.md'
+    with open(readme_correct_path, 'r') as readme_correct_file:
```

Comment:
very minor, you could do `readme_correct_path.open()`, Path objects allow this syntax. Tt is opened for text read by default so 'r' is not necessary.